lib/matplotlib/_mathtext.py

```diff
@@ -2841,6 +2859,40 @@ def binom(self, s, loc, toks):
         return self._genfrac('(', ')', 0.0, self._MathStyle.TEXTSTYLE,
                              num, den)
 
+    def _genset(self, state, value, symbol, shift):
```

Comment:
It's not super obvious which of `value` and `symbol` is the main part and which one is the "annotation", I guess we could rename them to e.g. `body` and `annotation` (throughout).